### ISSUE TITLE
Using 1.5.8

### ISSUE THREAD

**Issue by MajorReismann** [thetrueoneshots/CubeMegaMod#9]
Just wanted to share that when I use 1.5.8 I am completely stuck inside never ending buildings. version 1.5.7 works great. Just wanted to share this with you. I know it has a new world gen algorithm. Really Enjoying the mod.

**Comment by thetrueoneshots** [comment]
Yeah that is the worldgen update of 1.5.8 :D Therefore it is also a pre-release ^^
